join_group.html

```diff
@@ -0,0 +1,905 @@
+                                        <span class="cmp-button__text">Sign Up Now</span>
+                                    </a>
+                                </div>
+
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div
+                    class="container responsivegrid bottom-container aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="bottom-wrap" class="cmp-container">
+                        <div
+                            class="container responsivegrid bottom-content aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="container-d4e4b902f2" class="cmp-container">
+                                <div class="sub-title title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div id="title-5c2405ff2a" class="cmp-title">
+                                        <h6 class="cmp-title__text">GET STARTED</h6>
+                                    </div>
+                                </div>
+                                <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div id="title-c6c8e598f8" class="cmp-title">
+                                        <h3 class="cmp-title__text">Will a LeaderImpact group be a good fit
+                                            for me?</h3>
+                                    </div>
+                                </div>
+                                <div class="text aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <div id="text-924f8b967f" class="cmp-text">
+                                        <p>LeaderImpact groups are suitable for marketplace leaders,
+                                            business owners/directors, Industry leaders looking to develop
+                                            as leaders personally, professionally and spiritually.</p>
+                                    </div>
+                                </div>
+                                <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                    <a id="button-254edd7bf9" class="cmp-button" href="">
+                                        <span class="cmp-button__text">Sign Up Now</span>
+                                    </a>
+                                </div>
+
+                            </div>
+                        </div>
+                        <div
+                            class="container responsivegrid video-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-data-layer="{&quot;embed-2aa125dec4&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/embed&quot;}}"
```

Comment:
@Prystal254 This embed needs to be wrapped with `<div class="cmp-container"><div class="embed">...</div></div>`.